apps/e2ee-server/src/decorators/e2eeApiMethod.ts

```
@@ -0,0 +1,45 @@
+    target.constructor[E2EE_API_METHODS_KEY] = allowedMethods;
+
+    return descriptor;
+  };
+}
+
+/**
+ * Check if a method is allowed to be called via E2EE API
+ */
+export function isMethodAllowed(instance: any, methodName: string): boolean {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const constructor = instance.constructor;
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)